hw2/01-coins.js

```diff
@@ -2,6 +2,34 @@
 
 const calculateChange = (input) => {
   // Add your code here
+  if (input > 10) {
+    console.log("Error: the number is too large");
+  } else {
+    dollars = Math.floor(input % 10);
+    Remainder = (input % 1).toFixed(2);
+    quarter = Math.floor(Remainder / 0.25);
+    Remainder = (Remainder % 0.25).toFixed(2);
+    dimes = Math.floor(Remainder / 0.1);
+    Remainder = (Remainder % 0.1).toFixed(2);
+    nickel = Math.floor(Remainder / 0.05);
+    Remainder = (Remainder % 0.05).toFixed(2);
+    penny = Math.floor(Remainder / 0.01);
+    console.log(
```

Comment:
Hello,

The console.log statement may not print the correct string for plural currency. For example, in case there are multiple quarters as in sample test case "console.log(calculateChange(4.62));", it prints "4 dollar 2 quarter 1 dimes 0 nickel 2 penny", instead of "4 dollar 2 quarter(s) 1 dime 0 nickel 2 penn(ies).